src/composables/peerIdGenerator.js

```diff
@@ -0,0 +1,11 @@
+export function generatePeerIdCharacter() {
```

Comment:
Turn your code to 
`function generatePeerIdCharacter(){`
`let characters =`
   ` "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";`

  `let peerId = "";`

 ` for (let i = 0; i < 10; i++)`
   ` peerId += characters.charAt(Math.floor(Math.random() * characters.length));`

  `return peerId;`
`}`
`export {generatePeerIdCharacter};`

Reply:
I have made changes as requested, i appreciate your input.

However, l'd used the inline export because there is only a single function in the file. If it pleases you, i'd love to know why this implementation isn't accepted, so as not to make similar mistake in the future.

Reply:
Fixed